CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - fix: fix sharingan chain spec
 - fix: update madara infra to main branch
 - fix: update `Cargo.lock`
+- fix: exported chain id constant in mp-chain-id crate and added one for SN_MAIN
```

Comment:
Not a `fix`, rather a `refactor` 
https://www.conventionalcommits.org/en/v1.0.0-beta.4/#summary